src/System.Net.Security/src/System.Net.Security.csproj

```diff
@@ -83,6 +89,9 @@
     <Compile Include="$(CommonPath)\System\Net\InternalException.cs">
       <Link>Common\System\Net\InternalException.cs</Link>
     </Compile>
+    <Compile Include="$(CommonPath)\System\Collections\Generic\BidirectionalDictionary.cs">
+      <Link>Common\System\Collections\Generic\BidirectionalDictionary.cs</Link>
+    </Compile>
```

Comment:
Is this used by the Unix implementation?  If not, it should be in the Windows-specific section.